climada/util/test/test_coordinates.py

```diff
@@ -950,6 +950,33 @@ def test_bounded_refined_raster(self):
         self.assertLessEqual(transform[5] + (z.shape[0] - 0.5) * transform[4], bounds[1])
         self.assertGreater(transform[5] + (z.shape[0] - 1.5) * transform[4], bounds[1])
 
+        # trigger downloading of dist-to-coast dataset (if not already present)
+        path = u_coord.get_dist_to_coast_nasa_tif()
```

Comment:
Do you know if Jenkins will need to download this? Will he be OK with that?

Reply:
Yes, that's no problem. Jenkin's has this file permanently in cache and won't download it each time the tests are running, as far as I know. The reason is that this file is needed in other unit tests already, especially in the unit tests for the function `u_coord.dist_to_cast_nasa`:
https://github.com/CLIMADA-project/climada_python/blob/952f9ef9f60740babf0e124d4ca3d3d07e2dd292/climada/util/test/test_coordinates.py#L496-L498

Reply:
nice